src/actions.cpp

```diff
@@ -406,8 +407,58 @@ void evm_contract::pushtx( eosio::name miner, const bytes& rlptx ) {
     check(tx.max_priority_fee_per_gas == tx.max_fee_per_gas, "max_priority_fee_per_gas must be equal to max_fee_per_gas");
     check(tx.max_fee_per_gas >= current_config.gas_price, "gas price is too low");
 
+    // Filter EVM messages (with data) that are sent to the reserved address
+    // corresponding to the EOS account holding the contract (self)
+    ep.set_evm_message_filter([&](const evmc_message& message) -> bool {
+        auto me = make_reserved_address(get_self().value);
```

Comment:
You lost the `static` you had previously. Was this intentional? Does it do allocation? I want to make sure there isn't excessive work being done for each call of this filter function.